File: benchmark/corpus/csharp/cases/vuln_sqli.cs
using System.Data.SqlClient;

public class UserRepository
{
    public void LoadUser(SqlConnection connection)
    {
        var name = Request.QueryString["name"];
        var query = "SELECT * FROM Users WHERE Name = '" + name + "'";
        var command = new SqlCommand(query, connection); // ZS-CS-002: SQL injection
        command.ExecuteReader();
    }
}

File: benchmark/corpus/go/testdata/vuln_sqli.go
package main

import (
	"database/sql"
	"net/http"
)

func handler(db *sql.DB, r *http.Request) {
	id := r.URL.Query().Get("id")
	// ZS-GO-002: SQL injection — query built by concatenating tainted id
	query := "SELECT * FROM users WHERE id = " + id
	db.Query(query)
}
